test/unit/Cover/updateUintParameters.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai');
+
+describe('updateUintParameters', function () {
+  it('should revert if caller is not governance', async function () {
+    const { cover, accounts } = this;
+
+    await expect(cover.connect(accounts.nonMembers[0]).updateUintParameters([0], ['0'])).to.be.revertedWith(
+      'Caller is not authorized to govern',
+    );
+  });
+
+  it('should allow to update coverAssetsFallback', async function () {
+    const { cover, accounts } = this;
+
+    const coverAssetsFallbackBefore = await cover.coverAssetsFallback();
+    await cover.connect(accounts.governanceContracts[0]).updateUintParameters([2], ['1']);
+    const coverAssetsFallbackAfter = await cover.coverAssetsFallback();
+
+    expect(coverAssetsFallbackBefore).to.not.be.eq(coverAssetsFallbackAfter);
+  });
+
+  it('should allow to update globalRewardsRatio', async function () {
+    const { cover, accounts } = this;
+
+    const globalRewardsRatioBefore = await cover.globalRewardsRatio();
+    await cover.connect(accounts.governanceContracts[0]).updateUintParameters([1], ['1']);
+    const globalRewardsRatioAfter = await cover.globalRewardsRatio();
+
+    expect(globalRewardsRatioBefore).to.not.be.eq(globalRewardsRatioAfter);
```

Comment:
This assertion is not sufficient it only states the value changed but not to the value that was assigned.

Let's do a names constant for the actual value, and assert the. set value is equal to it.